src/EditorFeatures/CSharp/SplitComment/SplitCommentCommandHandler.CommentSplitter.cs

```diff
@@ -0,0 +1,92 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Linq;
+using System.Threading;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.Editor.Implementation.SplitComment;
+using Microsoft.CodeAnalysis.Formatting;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+using Microsoft.CodeAnalysis.Text;
+using static Microsoft.CodeAnalysis.Formatting.FormattingOptions;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.SplitComment
+{
+    internal partial class SplitCommentCommandHandler
+    {
+        private class CommentSplitter : AbstractCommentSplitter
+        {
+            internal const string CommentCharacter = "//";
```

Comment:
does this need to be internal (as opposed to private)?

Reply:
This needs to be internal as it's used outside the class